src/main/scala/com/lunatech/pointingpoker/actors/RoomManager.scala

```diff
@@ -0,0 +1,119 @@
+package com.lunatech.pointingpoker.actors
+
+import java.util.UUID
+
+import akka.actor.typed.scaladsl.{ActorContext, Behaviors}
+import akka.actor.typed.{ActorRef, Behavior, Terminated}
+import akka.actor.{ActorRef => UntypedRef}
+import com.lunatech.pointingpoker.actors
+import com.lunatech.pointingpoker.websocket.WSMessage
+import com.lunatech.pointingpoker.websocket.WSMessage.MessageType
+
+object RoomManager {
+
+  sealed trait Command
+  final case class CreateRoom(replyTo: ActorRef[Response])             extends Command
+  final case class IncomeWSMessage(message: WSMessage)                 extends Command
+  final case object UnsupportedWSMessage                               extends Command
+  final case class WSCompleted(roomId: UUID, userId: UUID)             extends Command
+  final case class WSFailure(t: Throwable)                             extends Command
+  final case class CompleteWS()                                        extends Command
+  final case class ConnectToRoom(message: WSMessage, user: UntypedRef) extends Command
+
+  sealed trait Response
+  final case class RoomId(value: String) extends Response
+
+  val InitialVoteState  = false
+  val InitialEstimation = ""
```

Comment:
I want to say that I had the same comment last time 😆 but imo this would be better as a `Option[String]` rather than just a `String`.

Reply:
I do agree that having as an Option would be better, however I'm choosing the simplicity here since this value is direct and only related to UI